code/modules/tgui_panel/external.dm

```diff
@@ -37,7 +37,7 @@
 	winset(src, "browseroutput", "is-disabled=0;is-visible=1")
 
 /client/verb/refresh_tgui()
-	set name = "Refresh TGUI"
+	set name = "Refresh Chat Interface"
```

Comment:
TGUI isn't just chat, just leave it, please. You can add a `(chat)` if it's really necesarry.